contracts/NFTStorefront.cdc

```diff
@@ -448,6 +551,38 @@ pub contract NFTStorefront {
             return self.listings.keys
         }
 
+        // getDublicateListingIDs
+        // Returns an array of listing Ids that are duplicate for the given `nftType` and `nftId`.
+        //
+        pub fun getDuplicateListingIDs(nftType: Type, nftID: UInt64, listingID: UInt64): [UInt64] {
+            let listingIds = self.listedNFTs[nftType][nftID]
+            // Verify that given listing Id also a part of the `listingIds`
+            let isListingExisted = listingIds.contains(listingID)
+            if isListingExisted {
+                var index: Int = 0
+                for id in listingIds {
+                    if id == listingId {
```

Comment:
```suggestion
                    if id == listingID {
```

i think this is meant to refer to the **listingID** that is the primary subject matter of the transaction, right? (for ex. the same listingID that is referenced on line 560)

might be good to standardize listingId vs listingID generally.